src/assets/images/logo-dark.svg

```diff
@@ -0,0 +1,54 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 608.57 613.79">
+  <defs>
+    <style>.cls-1 {
+      fill: url(#Blauegrün);
```

Comment:
Is `Baluegrün` a valid value? And what does `url(...)` do?

Reply:
Thats your svg. Don't know what you did there, but I fixed it.